### DISCUSSION TITLE
cifs shares, lxcs and service uids?

### DISCUSSION THREAD

**Discussion by millercentral** [tteck/Proxmox#3219] [Help Needed]
So this might be a dumb questions as I'm still learning, but I'm curious what others are doing.

I have a Proxmox server and I'm wanting to run both Plex and Jellyfin lxcs on it, configured by the helper scripts. 
My media is stored on a NAS, accessed via cifs. I mount the smb media share on the proxmox host, then map it to mount points to the two lxcs by adding mp0:... map point lines to the lxc configs. I include uid= and gid= mount options in the cifs mount command. This works great except that the UID/GID for the plex service account is different than the UID/GID for the jellyfin service account. So the mapped UID/GID on the smb share can only match one of them.

Is there a way during the lxc install to ensure a specific UID/GID for the service account of one or both of these lxc installs?

**Comment by asylumexp** [comment]
is there something preventing you from mounting them in the LXCs via cifs rather than the mount point?

**Reply by millercentral** [reply]
> is there something preventing you from mounting them in the LXCs via cifs rather than the mount point?

I want the mounts to be resilient to the nas rebooting or becoming temporarily unavailable, but systemd automounts units are not supported within an lxc. Also, somehow it feels more "efficient" to have a single connection to the NAS from the host that is shared to all the containers on the server.
